### PR TITLE
Add ttl parameter

### PR DESCRIPTION
**Pull request by slunak**
https://pushover.net/api#ttl

### CODE REVIEW

File: tests/Api/Message/MessageTest.php [hunk 1]
```diff
@@ -150,4 +150,40 @@ public function testGetTimestamp()
...
+    public function testGetTtl()
+    {
+        $message = new Message("This is a test message");
+
+        $message->setTtl(3600);
+
+        $this->assertEquals(3600, $message->getTtl());
+    }
+
+    public function testSetTtl()
```
Comment (OskarStark) [inline tests/Api/Message/MessageTest.php:163]: The test is exactly duplicate to the one above. Maybe rename the text to canSetAndGetTtl() ?

Comment (slunak) [reply tests/Api/Message/MessageTest.php:163]: Good catch @OskarStark . Would you be able to open an issue please and I can fix it later? Or feel free to fix it if you like. Thanks for reviewing the PR!

Comment (OskarStark) [reply tests/Api/Message/MessageTest.php:163]: Always happy to help 🙌

Comment (slunak) [reply tests/Api/Message/MessageTest.php:163]: Thank you :)

**Pull request merged**